examples/power_cycle/simple_example/loads_MAG.json

```diff
@@ -1,84 +1,50 @@
 {
-    "AFC": {
-        "name": "All Field Coils",
-        "reactive": {},
-        "active": {
-            "scc": {
-                "name": "superconducting coils consumption",
-                "module": null,
-                "variables_map": {
-                    "phases": [
-                        "dwl",
-                        "d2f",
-                        "ftt",
-                        "f2d"
-                    ],
-                    "normalize": [
-                        true,
-                        true,
-                        true,
-                        true
-                    ],
-                    "unit": "MW",
-                    "consumption": true,
-                    "efficiencies": {},
-                    "loads": {
-                        "description": [
-                            "toroidal, poloidal, central"
-                        ],
-                        "time": [
-                            "[0, 1]"
-                        ],
-                        "data": [
-                            "[5, 5]"
-                        ],
-                        "model": [
-                            "RAMP"
-                        ]
-                    }
-                }
-            }
+  "AFC": {
+    "name": "All Field Coils",
+    "reactive": {},
+    "active": {
+      "scc": {
+        "name": "superconducting coils consumption",
+        "module": null,
+        "variables_map": {
```

Comment:
All `normalize` JSON keys need to be exchanged by `normalise`, otherwise I don't think I can run the `simple.ex.py` example.

Reply:
The same is true of the `eudemo17.ex.py` example, but for some reason I cannot add review comments to that file.